app/src/main/res/layout/fragment_pledge_section_header_reward_sumary.xml

```diff
@@ -39,28 +39,28 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/pledge_header_title"
         app:layout_constraintEnd_toStartOf="@id/guideline_separator"
-        tools:text="Estimated delivery June 2020 " />
+        tools:text="Estimated delivery September 2020 " />
```

Comment:
Longest month to ensure that this TextView does not extend beyond the percentage